emr-etl-runner/Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM jruby:9.1
+MAINTAINER Josh Cox <josh 'at' webhosting.coop>
+
+ENV PLAIN_OF_JARS=20170705 \
+LANG=en_US.UTF-8
+
+RUN apt-get update \
+&& apt-get install -yqq gettext-base mlocate \
```

Comment:
thinking about it, I stuck em all in the same container because they share the same config file, which could be mounted in by yet another config container.